relay-server/src/actors/store.rs

```diff
@@ -15,8 +15,8 @@ use serde::{ser::Error, Serialize};
 
 use relay_common::{metric, ProjectId, UnixTimestamp, Uuid};
 use relay_config::{Config, KafkaTopic};
+use relay_general::protocol;
 use relay_general::protocol::{EventId, SessionStatus, SessionUpdate};
```

Comment:
_nit_: Please merge the imports:
```suggestion
use relay_general::protocol::{self, EventId, SessionStatus, SessionUpdate};
```

Reply:
done manually